prime-router/metadata/fhir_mapping/hl7/secondary/Contact.yml

```diff
@@ -13,7 +13,7 @@ name:
 
 telecom:
   condition: $contactPointXTN NOT_NULL
-  valueOf: datatype/ContactPoint
+  valueOf: datatypes/XTN/ContactPoint
```

Comment:
This change makes me a little nervous along with the change in order.yml since it covers ORC-23 which is an important field for STLTs. It's fine for HL7 senders but how will it work for SR on the outbound to HL7?

Reply:
We discussed this in a huddle, it's not super clear, but this is covered as part of the sender transform for SR, so this concern is addressed.